spec/jobs/search_keyword_job_spec.rb

```diff
@@ -6,7 +6,7 @@
   include ActiveJob::TestHelper
 
   describe '#perform' do
-    context 'given search engine is Google' do
+    context 'given Google is search engine' do
```

Comment:
This will always be true... at least as long as Google remains a search engine.

Should we keep the previous wording? Or maybe improve it now that search engines have been renamed `source`? :)

```suggestion
    context 'given the source is Google' do
```